sdk/src/main/java/dev/cdevents/events/ArtifactSignedCDEvent.java

```diff
@@ -86,17 +87,26 @@ public String currentCDEventType() {
 
     @Override
     public String schemaURL() {
-        return "https://cdevents.dev/0.3.0/schema/artifact-signed-event";
+        return "https://cdevents.dev/0.4.1/schema/artifact-signed-event";
+    }
+
+    /**
+    * @return the base URI of the schema
+    */
+
+    @Override
+    public String baseURI() {
+        return "https://cdevents.dev/0.4.1/schema/";
     }
 
 
     /**
-    * @return the artifactsigned.json schema Json
+    * @return the CDEvent's schema file name
     */
 
     @Override
-    public String eventSchema() {
-        return dev.cdevents.spec.schemas.SchemaLoader.loadSchema("artifactsigned.json");
+    public String schemaFileName() {
+        return "artifactsigned.json";
```

Comment:
Why was SchemaLoader changed to hardcode the 1st part of the schema location? As I recall it was made parameterizable for testing.

Reply:
SchemaLoader will not be used from now and it will load the schema from classpath by using JsonSchemaFactory itself  as below with `schemaMappers`
https://github.com/cdevents/sdk-java/pull/82/files#diff-5dd62731b665b7388a50d9a410891ba4ed8709bcd418c9782eacfeed6c6fe4f2R133-R138